File: Exemplos M10/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/BRIDGE/Commands/ConsultarUltimaTransacao.ts
import { BridgeCommand } from "../BridgeCommand";

export class ConsultarUltimaTransacao extends BridgeCommand {
  readonly pdv: string;

  constructor(pdv: string) {
    super("ConsultarUltimaTransacao");
    this.pdv = pdv;
  }

  functionParameters(): object {
    return { pdv: this.pdv };
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Bridge/ConsultarUltimaTransacao.cs
namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Bridge
{
    class ConsultarUltimaTransacao : BridgeCommand
    {
        private readonly string pdv;

        public ConsultarUltimaTransacao(string pdv) : base("ConsultarUltimaTransacao")
        {
            this.pdv = pdv;
        }

        protected override string FunctionParameters()
        {
            return "\"pdv\"" + ":" + "\"" + this.pdv + "\"";
        }
    }
}